mods/ra/languages/lua/en.ftl

```diff
@@ -0,0 +1,416 @@
+## campaign.lua
+objective-failed = Objective failed
+objective-completed = Objective completed
+
+primary = Primary
+secondary = Secondary
+
+new-primary-objective = New primary objective
+new-secondary-objective = New secondary objective
+
+## allies01, allies-03, allies-05, evacuation, personal-war
+tanya = Tanya
+
+## allies-01
+extract-einstein-helicopter = Wait for the helicopter and extract Einstein.
+find-einstein = Find Einstein.
+einstein-survive = Einstein must survive.
+protect-civilians = Protect all civilians.
+
+## allies-01, allies-03, allies-05, evacuation
+tanya-rules-of-engagement = According to the rules of engagement I need your explicit orders to fire, Commander!
+
+# allies-01, allies-05, evacuation
+tanya-survive = Tanya must survive.
+
+## allies-02, allies-05abc, sarin-gas-1-crackdown
+deny-allies = Deny the allies!
+
+## allies-02, soviet-06
+escort-convoy = Escort the convoy.
+
+## allies-02
+convoy-arrived = The convoy arrived!
+secure-convoy = Secure the convoy's path.
+eliminate-soviets = Eliminate the entire soviet presence in this area.
+
+## allies-03a
+destroy-oilpumps = Destroy all Soviet oil pumps.
+
+## allies-03b
+find-lost-tanks = Find our lost tanks.
+
+## allies-03ab
+destroy-bridges = Destroy all bridges.
+free-prisoners = Free all Allied soldiers and keep them alive.
+protect-bridges = Bridges must not be destroyed.
+
+## allies-04
+destroy-soviet-units-buildings = Destroy all Soviet units and buildings in this region.
+destroy-convoys = Eliminate all passing Soviet convoys.
+
+## allies-05a
+extract-transport = Get all your forces into the transport.
+
+# allies-05abc
+spy = Spy
+disguise-spy = Commander! You have to disguise me in order to get through the enemy patrols.
+battlefield-control = Battlefield Control
+skip-heroics = Good work! But next time skip the heroics!
+rescue-tanya = Rescue Tanya.
+elminate-soviet-units = Eliminate all Soviet units in this area.
+infiltrate-warfactory = Infiltrate the Soviet warfactory.
+destroy-sam-sites-blocker = Destroy all four SAM Sites that block
+    our reinforcements' helicopter.
+
+# allies-06ab
+soviet-tech-centers-discovered = Coordinates of the Soviet tech centers discovered.
+destroy-soviet-buildings-units = Destroy all Soviet buildings and units in the area.
+dont-capture-tech-centers = Do not capture the tech centers! Infiltrate one with a spy.
+infiltrate-tech-center-spy = Infiltrate one of the Soviet tech centers with a spy.
+capture-radar-shore = Capture the Radar Dome at the shore.
+
+# allies-07
+enemy-approaching = Enemy approaching
+soviet-armored-battalion-arrives-in = Soviet armored battalion arrives in{" "}
```

Comment:
That would mean translation lookup every tick.

Reply:
Oh I see. This is still wrong though.

We should introduce CachedTransforms

Reply:
A CachedTransform wouldn't solve everything, since we're still talking about a timer that is ticking down (so the value is changing quite often/regularly). In the end the performance impact is likely not that big that we could do the lookup, but I don't think that is necessary here.

The `{" "}` at the end feels a bit weird though. Why not remove that here and use `translation .. " " .. ticks` in Lua?

Reply:
Because other languages may want to put numbers in a different place. We aren't translating just for english

Reply:
Translators also need to know of the numbers existance